mautic/mautic
# [UX] Change "Google Analytics UTM tags" to remove the Google Analytics mention

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [x]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:

As concerns grow around the world regarding privacy or the way data is treated, there are several analysis tools created over time that use UTM (which would not be exclusively Google's).
I think it would be important for us to leave it open to represent the functionality without associating it with a specific tool.

![image](https://github.com/mautic/mautic/assets/116097999/847a7883-69a1-4b89-966a-47b364ac9e48)


- [x] Discussed with the community

> I actually like the idea of removing GA from the label. Tying UTM exclusively to Google helps maintain their monopoly on the market, which isn't ideal.

> I agree that just adding UTM tags is fine nowadays as I think most people understand what they are, and it's used by other tools than Google



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open Channels > Emails
3. Create new > select a type 
4. Look at the new label

![image](https://github.com/mautic/mautic/assets/116097999/c46d3e7d-7a35-4125-8dde-6026c48333b9)
